Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Spawning/DespawnNodeable.h

```diff
@@ -26,8 +26,9 @@ namespace ScriptCanvas::Nodeables::Spawning
         SCRIPTCANVAS_NODE(DespawnNodeable);
     public:
         DespawnNodeable() = default;
-        DespawnNodeable(const DespawnNodeable&) = delete;
-        DespawnNodeable& operator=(const DespawnNodeable&) = delete;
+        // these methods are required by Script Canvas
+        DespawnNodeable(const DespawnNodeable&) {}
+        DespawnNodeable& operator=(const DespawnNodeable&) {}
```

Comment:
I'd think these should be `= default;` to implement the actual functionality, no?

Reply:
+1, some more context would be nice to have here. I have a few questions:
1. You say "copy ctor and assignment are required by Script Canvas after all". How did you find that?
2. Are copy operations needed so that every node in SC can be clicked and ctrl+C/ctrl+v should be possible?
3. Is it common practice for script canvas nodes to have empty copy operations like this?

Reply:
no it's needed when entering game mode. It is called by serializer I believe. There's an error in console when they are missing and nodes wont execute. I don't know about common practice, we are pioneers in this area and there aren't many examples to look at, but we also have a custom use case where some members should NOT be copied such as spawnable tickets.

I've updated the CR with better comments.

Reply:
So, ctrl+g is requiring a copy of SC nodes? That doesn't sound like a correct behavior. It might be good idea for @AMZN-koppersr to take a look at this as it feels like we are ignoring an underlying issue here.